index.js

```diff
@@ -34,7 +34,7 @@ const N_DIV_2 = new BN('7fffffffffffffffffffffffffffffff5d576e7357a4501ddfe92f46
  * @prop {Buffer} s EC recovery ID
  */
 module.exports = class Transaction {
-  constructor (data) {
+  constructor (data, chainId) {
```

Comment:
Can we make this a hashmap instead? e.g.:

```
constructor (data, opts) {
```

So that if more options will surface in the future we don't need to add new variables and is also a bit more readable:
```

```
var eip155tx = new Tx(rawTx, { chain_id: 1})
```